index.js

```diff
@@ -123,7 +123,9 @@ alfy.fetch = (url, opts) => {
 
 	const rawKey = url + JSON.stringify(opts);
 	const key = rawKey.replace(/\./g, '\\.');
-	const cachedResponse = alfy.cache.store[rawKey] && alfy.cache.store[rawKey].data;
+
+	const cachedData = (alfy.cache.store[rawKey] && alfy.cache.store[rawKey].data) || {};
+	const cachedResponse = cachedData.version === alfy.meta.version ? cachedData.data : undefined;
```

Comment:
I don't think we should use `semver` here, right? It has to be an exact match imo.